### PR TITLE
Added type hints to additional tests

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

File: Tests/test_imagecms.py [hunk 1]
```diff
@@ -335,12 +335,26 @@ def test_extended_information() -> None:
...
     p = o.profile
 
-    def assert_truncated_tuple_equal(tup1, tup2, digits: int = 10) -> None:
+    def assert_truncated_tuple_equal(
+        tup1: tuple[tuple[float, float, float], ...] | tuple[float],
+        tup2: (
+            tuple[tuple[tuple[float, float, float], ...], ...]
+            | tuple[tuple[float, float, float], ...]
+            | tuple[float]
+        ),
```
Comment (hugovk) [inline Tests/test_imagecms.py:344]: Probably okay to make use of `tuple[Any]` or even `Any` here and in `truncate_tuple` to simplify things, but I don't mind too much :)

**Review by hugovk — APPROVED** [review]

File: Tests/test_imagecms.py (hunk 1 above)
Comment (radarhere) [reply Tests/test_imagecms.py]: Ok, I've pushed a commit with `tuple[Any, ...]`

File: Tests/test_imagecms.py [hunk 2]
```diff
@@ -336,25 +337,13 @@ def test_extended_information() -> None:
...
         power = 10**digits
 
-        def truncate_tuple(
-            tuple_or_float: (
-                tuple[tuple[tuple[float, float, float], ...], ...]
-                | tuple[tuple[float, float, float], ...]
-                | tuple[float, ...]
-            )
-        ) -> tuple[tuple[float, ...], ...]:
+        def truncate_tuple(tuple_or_float: tuple[Any, ...]) -> tuple[Any, ...]:
```
Comment (hugovk) [inline Tests/test_imagecms.py:346]: This parameter is called `tuple_or_float` but typed only as a `tuple`. 

Do we ever call it as a `float`? If not, shall we rename it?

Comment (radarhere) [reply Tests/test_imagecms.py]: I've pushed a commit to rename it to `tuple_value`.

**Pull request merged**